ML/PYTORCH/README.md

```diff
@@ -0,0 +1,33 @@
+This tutorial presents an example of using a pre-trained Pytorch model in the AMReX framework.
+A model is included as an example (`/Exec/model.pt`).
+
+You will first need to download the appropriate Pytorch library (libtorch) in the current directory. Note that we have renamed the folder to indicate whether it uses CPU or CUDA.
+
+```shell
+# CPU only
+wget https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.9.0%2Bcpu.zip
+unzip libtorch-cxx11-abi-shared-with-deps-1.9.0+cpu.zip
+mv libtorch libtorch_cpu
+
+# CUDA 11.1
+wget https://download.pytorch.org/libtorch/cu111/libtorch-cxx11-abi-shared-with-deps-1.9.0%2Bcu111.zip
+unzip libtorch-cxx11-abi-shared-with-deps-1.9.0%2Bcu111.zip
+mv libtorch libtorch_cuda
+
+# CUDA 10.2
+wget https://download.pytorch.org/libtorch/cu111/libtorch-cxx11-abi-shared-with-deps-1.9.0%2Bcu102.zip
```

Comment:
This doesn't work for me - says "ERROR 405: Forbidden"

Reply:
oops, that's another typo